3rd-party-licenses.md

```diff
@@ -50,98 +52,174 @@ https://github.com/selectize/selectize.js/blob/master/LICENSE
+License: Creative Commons Attribution 3.0 BY
+https://github.com/erikdubbelboer/phpRedisAdmin/blob/master/README.markdown
+
+### Selenium Docker
+Image: Selenium/standalone-chrome:3.11  
+License: Apache License 2.0  
+https://github.com/SeleniumHQ/docker-selenium/blob/master/LICENSE.md
+
+### Adminer
+Image: Adminer:4.6  
+License: Apache License 2.0 or GPL 2  
+https://github.com/vrana/adminer/blob/master/readme.txt
+
+### Elasticsearch
+Image: Docker.elastic.co/elasticsearch/elasticsearch-oss  
+License: Apache License 2.0  
+https://github.com/elastic/elasticsearch/blob/66b5ed86f7adede8102cd4d979b9f4924e5bd837/LICENSE.txt
+
+### Php
+Image: php:7.2-fpm-alpine  
+License: The PHP License  
+http://php.net/license/
+
+### GNU libiconv
+Package: gnu-libiconv  
+License: LGPL  
+https://pkgs.alpinelinux.org/package/edge/testing/x86/gnu-libiconv
+
+### Composer - Dependency Management for PHP
+License: MIT
+https://github.com/composer/composer/blob/master/LICENSE
+
+### grunt-cli
+License: MIT  
+https://github.com/gruntjs/grunt-cli/blob/master/LICENSE-MIT
+
 ### nodejs-npm
 License: Artistic License 2.0  
 https://www.npmjs.com/policies/npm-license
 
-### gnu-libiconv
-License: GPL v3  
-https://directory.fsf.org/wiki/Libiconv
-
-### eslint-config-standard
+### prestissimo (composer plugin)
 License: MIT  
-https://github.com/standard/eslint-config-standard/blob/master/LICENSE
+https://github.com/hirak/prestissimo/blob/master/LICENSE
 
-### Autoprefixer
-License: MIT  
-https://github.com/postcss/autoprefixer/blob/master/LICENSE
+### libpng-dev
+License: GPL
+https://pkgs.alpinelinux.org/package/v3.3/main/x86/libpng-dev
 
-### ES6-Promise (subset of rsvp.js)
-License: MIT  
-https://github.com/stefanpenner/es6-promise/blob/master/LICENSE
+### icu-dev
+License: MIT ICU Unicode-TOU  
+https://pkgs.alpinelinux.org/package/edge/main/x86/icu-dev
 
-### time-grunt
-License: MIT  
-https://github.com/sindresorhus/time-grunt/blob/master/license
+### postgresql-dev
+License: PostgreSQL  
+https://pkgs.alpinelinux.org/package/edge/main/x86/postgresql-dev
+
+### libzip-dev
+License: BSD-3-clause  
+https://pkgs.alpinelinux.org/package/edge/community/x86/libzip-dev
+
+### autoconf
+License: GPL2+  
+https://pkgs.alpinelinux.org/package/v3.3/main/x86/autoconf
+
+### freetype-dev
+License: FTL GPL2+  
+https://pkgs.alpinelinux.org/package/edge/main/x86/freetype-dev
+
+### libjpeg-turbo-dev
+License: GPL  
+https://pkgs.alpinelinux.org/package/edge/main/x86/libjpeg-turbo-dev
+
+### pecl
+License: The PHP License  
+https://pecl.php.net/copyright.php
+
+### postgresql
+License: PostgreSQL  
+https://pkgs.alpinelinux.org/package/edge/main/x86/postgresql-dev
+
+## Others css and js libraries
+Other components, mostly css and js libraries, that are not dynamically installed.
+They can be found primarily in the `ShopBundle/Resources/` and `FrameworkBundle/Resources/` directories.
 
-### jit-grunt 0.10.0
+### Magnific Popup Repository
```

Comment:
Would it make sense to provide also the path to that file / directory? It would make it much more searchable and maintainable

Reply:
Done